pkg/util/fs.go

```diff
@@ -77,39 +77,77 @@ func (h *fs) Exists(file string) bool {
 	return err == nil
 }
 
-// Copy copies a set of files from sourcePath to targetPath
-func (h *fs) Copy(sourcePath string, targetPath string) error {
-	if _, err := os.Stat(sourcePath); err != nil {
+// Copy copies the source file to a destination directory.
+// If the destination is a filename, the file will be replaced.
+// If the source is directory, the directory itself and also the content of the
+// source directory will be copied into a target directory.
+// If the destination is a directory an error will be returned as the
+// destination has to be full path to a file.
```

Comment:
@bparees 

`Copy()` copies file or a directory into a destination. If the source is file, then the destination has to be file as well. If the source is a directory, then we use CopyContents().